RELEASE.md

```diff
@@ -1,14 +1,59 @@
-# Stacks Release Process
+# Appsody Stacks Release Process
+A release in this repo represents a new version of a particular stack - denoted by the release tag. It will contain the following assets:
+- archives for all the templates for the released stack on [Appsody stacks GitHub releases](https://github.com/appsody/stacks/releases)
+- latest indexes for each of the repos (`incubator`, `experimental`, `stable`) on [Appsody stacks GitHub releases](https://github.com/appsody/stacks/releases)
+- updated docker hub images with appropriate tags on [Appsody dockerhub](https://hub.docker.com/u/appsody)
```

Comment:
nit: docker hub -> DockerHub (consistency with line 53)

Reply:
@skoh7645 this got merged do you want to submit a PR for this?